input/pagecontent/index.md

```diff
@@ -1,102 +1,88 @@
 
-[Spirometry](https://en.wikipedia.org/wiki/Spirometry) measure the volume and flow of air that the lungs can inhale or exhale. This includes:
-- Forced vital capacity (FVC) - the total volume of air which can be exhaled after taking as deep a breath as possible
-- Forced expiratory volume (FEV) - the volume of air which can be forcibly exhaled over some number of seconds
-- FEV1/FVC - the ratio of forced expiratory volume over 1 second to forced vital capacity
-- Forced expiratory time (FET) - the amount of time it takes to fully exhale after taking as deep a breath as possible
+Additionally, they note a number of significant organizational challenges to overcome in driving interoperability for PFTs: "PFT data [has not been] prioritized for standardization by organizational bodies (FHIR)", "lack of consensus on which discrete variables should be sent to the EHR", and "lack of common data model and interoperability standard by societies".
 
-### Diffusing Capacity
-[Diffusing capacity](https://en.wikipedia.org/wiki/Diffusing_capacity) measures the ability of the lungs to transfer gas from air to the blood vessels in the lungs.
-This includes:
-- TODO
+Finally, they point towards the benefit of a standardized data vocabulary for PFTs:
 
-Spirometry is by far the most common test panel included in pulmonary function tests, followed by diffusing capacity.
+> Availability of a standardized data vocabulary that labels and defines PFT elements would benefit both manufacturers of PFT equipment that generates pulmonary function data and developers of EHR software that receives the data and manages it. Moving from proprietary solutions to standardized information models and approaches to data exchange between PFT equipment and EHRs would have many benefits. It would promote interoperability, enabling data generated by pulmonary function equipment from a wide range of manufacturers to be recognized by a wide range of EHR systems and to be shared between different brands of EHR systems.
 
-## Background
+This IG is a response to this articulation.
 
-TODO
+### Usage
 
-## Motivation
+A `DiagnosticReport` profile with an identifier of `PulmonaryFunctionTestDiagnosticReport` is introduced.
 
-TODO
+This implementation guide can be used by device manufacturers, pulmonary function labratories, and electronic health record vendors to model pulmonary function test data.
+The `PulmonaryFunctionTestDiagnosticReport` defines a set of reference `Observation` Profiles that map to the 2017 [Recommendations for a Standardized Pulmonary Function Report](https://www.thoracic.org/statements/resources/pft/standardized-pulmonary-function-report.pdf). As a design principle, this implementation guide makes use of `derivedFrom` associations for `Observation` Profiles of measures such as the Zscore of a patient's performance against a reference value equation. Where possible, this implementation guide.
```

Comment:
>Where possible, this implementation guide.

Did you omit something here, or is this to be removed/expanded on?

Reply:
>As a design principle, this implementation guide makes use of derivedFrom associations for Observation Profiles of measures such as the Zscore of a patient’s performance against a reference value equation. 

Technically, this is not true for the Profiles, only the example FVC Instances. But I think it would be good to slice `derivedFrom` in the Zscore and PercentOfPredicted Profiles now that we've gotten the IG build process working and have fleshed out the documentation.

So I think leave this for now, and if we end up not slicing `derivedFrom` then we can specify we provide examples of how a user might perform that.

Reply:
```suggestion
The `PulmonaryFunctionTestDiagnosticReport` defines a set of reference `Observation` Profiles that map to the 2017 [Recommendations for a Standardized Pulmonary Function Report](https://www.thoracic.org/statements/resources/pft/standardized-pulmonary-function-report.pdf). This implementation contains `Instance` examples where `derivedFrom` associations for `Observation` are used on measures such as the Zscore of a patient's performance against a reference value equation. 
```